web-extension/search.js

```diff
@@ -121,26 +127,39 @@ function _onSearchResults(response, isHostQuery) {
     searching = false;
 }
 
-function _onResultSelected(event, element) {
-    element = element || event.target;
-    const value = element.innerText;
+function _onEntryAction(event, element) {
+    if (element) {
+        event.target = element;
+    }
+
     if (event.altKey) {
-        sendNativeAppMessage({ type: 'getData', entry: value }).then(
-            message => onEntryData(element, message),
-            logAndDisplayError
-        );
+        _onEntryDetails(event);
     } else if (event.shiftKey) {
-        sendNativeAppMessage({ type: 'getLogin', entry: value }).then(
-            _onLoginCredentialsDoCopyClipboard,
-            logAndDisplayError
-        );
+        _onEntryCopy(event);
     } else {
+        const value = element.innerText;
```

Comment:
I'm getting an error here when trying to login `search.js, line 140: TypeError: element is undefined`